src/libraries/System.Drawing.Common/src/misc/GDI/WindowsGraphics.cs

```diff
@@ -57,50 +57,37 @@ public static WindowsGraphics FromGraphics(Graphics g, ApplyGraphicsProperties p
 
             PointF offset = default;
 
-            Region? clipRgn = null;
-            Matrix? worldTransf = null;
-
-            if ((properties & ApplyGraphicsProperties.TranslateTransform) != 0 || (properties & ApplyGraphicsProperties.Clipping) != 0)
+            if (properties != ApplyGraphicsProperties.None)
             {
-                if (g.GetContextInfo() is object[] data && data.Length == 2)
+                Region? clip = null;
+
+                if (properties.HasFlag(ApplyGraphicsProperties.Clipping))
```

Comment:
Don't we avoid using `HasFlag`?

Reply:
@RussKie is correct. And that was the only reason.